<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MicsGeocodePluginDialog</class>
 <widget class="QDialog" name="MicsGeocodePluginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MicsGeocodePlugin</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_step1">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Step 1</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>451</width>
          <height>561</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QLineEdit" name="basenameLineEdit">
           <property name="text">
            <string>project_basename</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="outputDirLineEdit">
             <property name="text">
              <string>output_directory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="outputDirToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxCentroid">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="title">
            <string>Centroid Source</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>451</width>
              <height>211</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayoutCSF">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="centroidsSourceFileLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Centroid Source File</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="centroidsSourceFileLineEdit"/>
               </item>
               <item>
                <widget class="QToolButton" name="centroidsSourceFileToolButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QStackedWidget" name="stackFields">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="pageAuto">
                <widget class="QWidget" name="layoutWidget">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>431</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="typeFieldLabel_3">
                    <property name="text">
                     <string>Type Field</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="typeFieldLineEdit">
                    <property name="text">
                     <string>Type</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="numeroFieldLabel">
                    <property name="text">
                     <string>Numero Field</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="numeroFieldLineEdit">
                    <property name="text">
                     <string>Numero</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="longitudeFieldLabel_3">
                    <property name="text">
                     <string>Longitude Field</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="longitudeFieldLineEdit">
                    <property name="text">
                     <string>Long</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="latitudeFieldLabel_3">
                    <property name="text">
                     <string>Latitude Field</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="latitudeFieldLineEdit">
                    <property name="text">
                     <string>Lat</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
               <widget class="QWidget" name="pageManual">
                <widget class="QWidget" name="layoutWidget">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>431</width>
                   <height>141</height>
                  </rect>
                 </property>
                 <layout class="QFormLayout" name="formLayout_3">
                  <item row="0" column="0">
                   <widget class="QLabel" name="typeFieldLabel_2">
                    <property name="text">
                     <string>Type</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="typeFieldComboBox"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="numeroFieldLabel_2">
                    <property name="text">
                     <string>Numero Field</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QComboBox" name="numeroFieldComboBox"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="longitudeFieldLabel_2">
                    <property name="text">
                     <string>Longitude Field</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QComboBox" name="longitudeFieldComboBox"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="latitudeFieldLabel_2">
                    <property name="text">
                     <string>Latitude Field</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QComboBox" name="latitudeFieldComboBox"/>
                  </item>
                 </layout>
                </widget>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="loadCentroidsButton">
               <property name="text">
                <string>Load centroids</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="title">
            <string>Reference Layer</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>23</y>
              <width>451</width>
              <height>171</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayoutRL">
               <property name="sizeConstraint">
                <enum>QLayout::SetNoConstraint</enum>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="referenceLayerLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Reference Layer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="referenceLayerLineEdit"/>
               </item>
               <item>
                <widget class="QToolButton" name="referenceLayerToolButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="referenceLayerFieldCombobox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="ruralValuesLabel">
                 <property name="text">
                  <string>Rural Values</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="ruralValuesLineEdit">
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string>R</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="urbanValuesLabel">
                 <property name="text">
                  <string>Urban Values</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="urbanValuesLineEdit">
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string>U</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="displaceCentroidsButton">
           <property name="text">
            <string>Displace Centroids</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_step2">
       <attribute name="title">
        <string>Step 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="loadConfigButton">
      <property name="text">
       <string>Load Config</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveConfigButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveConfigAsButton">
      <property name="text">
       <string>Save Config as</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
